src/layouts/BaseLayout.astro

```diff
@@ -139,6 +140,7 @@ if (isFallback) canonicalURL.pathname = canonicalURL.pathname.replace(`/${lang}/
 			</aside>
 			<div id="main-content" class="main-column">
 				<slot />
+				<span id="docsearch-lvl0" hidden>{getPageCategory(url)}</span>
```

Comment:
I wanted to test out if the crawler picks this up as expected but it’s only able to crawl on our domain, so doesn’t work on the Netlify preview. I think it should, but if it doesn’t I’ll remove this and try a different approach.